### DISCUSSION TITLE
Informational finding marked as medium

### DISCUSSION THREAD

**Discussion by BiasedMerc** [solodit/feedback#52] [Bug Report]
Solodit finding page: https://solodit.xyz/issues/m-1-erc20-return-values-not-checked-sherlock-derby-derby-git
Sherlock github finding page: https://github.com/sherlock-audit/2023-01-derby-judging/issues/10
The finding was marked as informational (but auditor initially submitted it as medium) however solodit is showing it as a medium.

**Comment by wang6131** [comment]
@BiasedMerc 
Thank you for your report.
Fixed the severity to Low.
